python/python/res/server/ertrpcserver.py

```diff
@@ -39,24 +40,14 @@ def createFault(error, message):
     error_code = INVERSE_FAULT_CODES[error]
     return Fault(error_code, message)
 
-
-
 class ErtRPCServer(SimpleXMLRPCServer):
-    def __init__(self, config, host="localhost", port=0, log_requests=False, verbose_queue=False):
+    def __init__(self, enkf_main=None, res_config=None, host="localhost", port=0, log_requests=False, verbose_queue=False):
```

Comment:
Should maybe make it clearer from the api which of the arguments, i.e. `enkf_main` and `res_config` which is *preferred*?